crud.py

```diff
@@ -0,0 +1,66 @@
+from sqlalchemy.orm import Session, joinedload
+
+from models import DBBook, DBAuthor
+from schemas import AuthorCreate, BookCreate
+
+
+def get_all_authors(
+    db: Session,
+    skip: int = 0,
+    limit: int = 10,
+) -> list[DBAuthor]:
+    return (db.query(DBAuthor)
+            .options(joinedload(DBAuthor.books))
```

Comment:
The use of joinedload is great for optimizing the loading of related data (e.g., loading books with authors). However, ensure that you really need to load all related data in every query, as it can add overhead if the related tables are large.